serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -143,34 +139,41 @@ For more information, see <<proc-install-knative-minikube, Installing Knative on
 ----
 eval $(minikube -p minikube docker-env --profile knative)
 ----
+--
 
-When using Minikube, you can share your local registry with the Docker Daemon.
-
-NOTE: If your Serverless Workflow application container image was built before configuring Docker, then you might need to build the image again so that the image is available on the Minikube Docker.
+. Build your application to store it into the Minikube registry
++
+--
+If your Serverless Workflow application container image was built before configuring Docker to use in-cluster Docker daemon, then you might need to build the image again so that the image is available in the Minikube registry.
```

Comment:
```suggestion
If your workflow application container image is built before configuring Docker to use the in-cluster Docker daemon, then you might need to build the image again so that the image is available in the Minikube registry.
```